src/python_inspector/api.py

```diff
@@ -103,9 +103,6 @@ def resolve_dependencies(
             f"Must be one of: {', '.join(PLATFORMS_BY_OS.keys())}"
         )
 
-    valid_python_versions = list(PYTHON_DOT_VERSIONS_BY_VER.keys())
```

Comment:
This would be an API breaking change and I am not sure I get the value for this

Reply:
https://github.com/nexB/python-inspector/pull/109/commits/3a66ca2e30d8fd6e668219317723843f138145ab Removed this stuff which were not part of this PR